(PDB-5412) Add release notes for 6.20.0 (#3619)
puppetlabs/puppetdb: documentation/release_notes_6.markdown

# PuppetDB: Release notes

## PuppetDB 6.20.0

Released January 20 2022

### New features and improvements

* Improved performance of the "deactivate node" command. ([PDB-5378](https://tickets.puppetlabs.com/browse/PDB-5378))
* Improved performance of the fact-contents endpoint. Testing against a database of 10,000 mocked nodes, there was an observed 84% decrease in time taken to complete a difficult query. ([PDB-5259](https://tickets.puppetlabs.com/browse/PDB-5259)

### Bug Fixes

> * Fixed a bug with HA sync (Puppet Enterprise only) regarding `/pdb/query/v4/<entity>/<certname>` style queries that caused replicas to falsely report a node count of 0. ([PDB-5381](https://tickets.puppetlabs.com/browse/PDB-5381))

Reviewer: ```suggestion
* Fixed a bug with HA sync (Puppet Enterprise only) regarding `/pdb/query/v4/<entity>/<certname>` style queries that caused replicas to falsely report that the sync transferred 0 nodes. ([PDB-5381](https://tickets.puppetlabs.com/browse/PDB-5381))
```
